CHANGELOG.txt

```diff
@@ -36,6 +36,8 @@ master:
  - obspy.scripts:
    * obspy-scan command line script now also plots and prints overlaps
      alongside gaps (see #1366)
+ - obspy.io.gcf
```

Comment:
Please keep in alphabetical order. But actually, new format support should go in the General section (see the 1.0.0 entry for an example.)